Umbraco-Heartcore/Versions-and-updates/index.md

```diff
@@ -0,0 +1,17 @@
+---
+versionFrom: 8.0.0
+---
+
+# Versions and updates
+
+Umbraco Heartcore is a headless SaaS (software as a service) product, which means that one of the benefits is that you do not have to worry about upgrades. These will all be handled by Umbraco HQ. 
+
+Umbraco HQ will regularly be adding new features and endpoints to the REST APIs, as well as maintaining the existing structure and endpoints. Each time this is done, a new version of the API will be released. API releases and versioning is handled following the [ASP.NET API versioning guidelines](https://github.com/microsoft/aspnet-api-versioning).
+
+:::note
+It is important that you **use the `API-version` header to specify which version of the API you will be using** when making calls to the API. If no version is specified the latest vesrion of the API will be used, which could potentially break your client if it is setup for a lower version.
```

Comment:
When making calls to the REST API we recommend that you **use the `api-version` header to specify which version of the API you will be using**. If no `api-version` is specified the latest version of the API will be used.

When using the client libraries the `api-version` is handled as part of the library and generally not something you need to worry about. Just be aware that the client library releases will have certain features available - as an example, if you are using the very first release (1.0) of the .net core client library it will not have the Forms API available. That was made available in the 1.1 version release of the client library.